accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -0,0 +1,475 @@
+    - `get_DataBits()`
+    - `get_DiscardNull()`
+    - `get_DsrHolding()`
+    - `get_DtrEnable()`
+    - `get_Encoding()`
+    - `get_Handshake()`
+    - `get_IsOpen()`
+    - `get_NewLine()`
+    - `get_Parity()`
+    - `get_ParityReplace()`
+    - `get_PortName()`
+    - `get_ReadBufferSize()`
+    - `get_ReadTimeout()`
+    - `get_ReceivedBytesThreshold()`
+    - `get_RtsEnable()`
+    - `get_StopBits()`
+    - `get_WriteBufferSize()`
+    - `get_WriteTimeout()`
+    - `GetPortNames()`
+    - `Open()`
+    - `Read(Byte[], Int32, Int32)`
+    - `Read(Char[], Int32, Int32)`
+    - `ReadByte()`
+    - `ReadChar()`
+    - `ReadExisting()`
+    - `ReadLine()`
+    - `ReadTo(String)`
+    - `set_BaudRate(Int32)`
+    - `set_BreakState(Boolean)`
+    - `set_DataBits(Int32)`
+    - `set_DiscardNull(Boolean)`
+    - `set_DtrEnable(Boolean)`
+    - `set_Encoding(Encoding)`
+    - `set_Handshake(Handshake)`
+    - `set_NewLine(String)`
+    - `set_Parity(Parity)`
+    - `set_ParityReplace(Byte)`
+    - `set_PortName(String)`
+    - `set_ReadBufferSize(Int32)`
+    - `set_ReadTimeout(Int32)`
+    - `set_ReceivedBytesThreshold(Int32)`
+    - `set_RtsEnable(Boolean)`
+    - `set_StopBits(StopBits)`
+    - `set_WriteBufferSize(Int32)`
+    - `set_WriteTimeout(Int32)`
+    - `Write(Byte[], Int32, Int32)`
+    - `Write(Char[], Int32, Int32)`
+    - `Write(String)`
+    - `WriteLine(String)`
+
+### System.Net
+
+* **HttpListenerTimeoutManager**
+    - `set_EntityBody(TimeSpan)`
+    - `set_HeaderWait(TimeSpan)`
+    - `set_MinSendBytesPerSecond(Int64)`
+    - `set_RequestQueue(TimeSpan)`
+* **IPEndPoint**
+    - `Create(SocketAddress)`
+* **SocketAddress**
+    - `.ctor(AddressFamily)`
+    - `.ctor(AddressFamily, Int32)`
+    - `get_Family()`
+    - `ToString()`
+
+### System.Net.Sockets
+
+* **Socket**
+    - `AcceptAsync(SocketAsyncEventArgs)`
+    - `BeginAccept(Int32, AsyncCallback, Object)`
+    - `BeginAccept(Socket, Int32, AsyncCallback, Object)`
+    - `BeginReceiveFrom(Byte[], Int32, Int32, SocketFlags, EndPoint, AsyncCallback, Object)`
+    - `BeginReceiveMessageFrom(Byte[], Int32, Int32, SocketFlags, EndPoint, AsyncCallback, Object)`
+    - `IOControl(Int32, Byte[], Byte[])`
+    - `IOControl(IOControlCode, Byte[], Byte[])`
+    - `ReceiveFrom(Byte[], EndPoint)`
+    - `ReceiveFrom(Byte[], Int32, Int32, SocketFlags, EndPoint)`
+    - `ReceiveFrom(Byte[], Int32, SocketFlags, EndPoint)`
+    - `ReceiveFrom(Byte[], SocketFlags, EndPoint)`
+    - `ReceiveFromAsync(SocketAsyncEventArgs)`
+    - `ReceiveMessageFrom(Byte[], Int32, Int32, SocketFlags, EndPoint, IPPacketInformation)`
+    - `ReceiveMessageFromAsync(SocketAsyncEventArgs)`
+    - `SetIPProtectionLevel(IPProtectionLevel)`
+* **SocketTaskExtensions**
+    - `AcceptAsync(Socket)`
+    - `AcceptAsync(Socket, Socket)`
+    - `ReceiveFromAsync(Socket, ArraySegment<Byte>, SocketFlags, EndPoint)`
+    - `ReceiveMessageFromAsync(Socket, ArraySegment<Byte>, SocketFlags, EndPoint)`
```

Comment:
All of the above are supported on unix.

AcceptAsync(Socket, Socket) is more nuanced, in that it works fine on Unix as long as the second Socket argument is null.